packages.ini

```diff
@@ -707,6 +742,7 @@ validate_incorrect_missing_deps = beautifulsoup4
 
 [uritemplate==4.1.1]
 
+[urllib3==1.26.6]
```

Comment:
I happen to know this version has a vulnerability

Reply:
link? couldn't find anything on it on a quick search